modin/pandas/test/test_io.py

```diff
@@ -3095,6 +3097,12 @@ def test_read_xml(self):
    <degrees>360</degrees>
    <sides/>
  </row>
+ <row>
```

Comment:
Why is this added?

Reply:
The example is outdated, I took the updated one from https://pandas.pydata.org/docs/reference/api/pandas.read_xml.html#pandas-read-xml.